internal/services/rewards_test.go

```diff
@@ -806,6 +806,139 @@ func TestBaselineIssuance(t *testing.T) {
 	}
 }
 
+func TestSyntheticDeviceIssuance(t *testing.T) {
+	ctx := context.Background()
+
+	settings, err := shared.LoadConfig[config.Settings]("settings.yaml")
+	if err != nil {
+		t.Fatal(err)
+	}
+	settings.TransferBatchSize = 2
+
+	logger := zerolog.Nop()
+
+	cont, conn := utils.GetDbConnection(ctx, t, logger)
+	defer func() {
+		err := cont.Terminate(ctx)
+		assert.NoError(t, err)
+	}()
+
+	scens := []Scenario{
```

Comment:
suggestion:
- add test that includes different combinations:
          - two synthetic devices --> only connection should earn
          - synthetic and aftermarket --> if valid pairing (ie not tesla and autopi), earn expected total

Reply:
I think we've covered your second scenario in the above suite. The first one would be good to see—can we add such a test? Something in which we've seen signals from both Smartcar and Tesla, but only the Tesla synthetic device is paired.

To be honest, it's not clear to me that this should be in a separate suite. Is the shape any different from the earlier tests?